tgui/packages/tgui/interfaces/PipeDispenser.js

```diff
@@ -0,0 +1,102 @@
+import { useBackend, useLocalState } from '../backend';
+import { Button, Section, Flex, Box, NumberInput } from '../components';
+import { Window } from '../layouts';
+
+export const PipeDispenser = (props, context) => {
+  const { act, data } = useBackend(context);
+  const disposalpipes = data.disposalpipes || [];
+  const [amount, setAmount] = useLocalState(context, 'amount', 1);
+  const {
+    dispenser_ready,
+    windowName,
+    mobile,
+    removing_pipe,
+    laying_pipe,
+    max_disposal_pipes,
+  } = data;
+  return (
+    <Window
+      title={windowName}
+      width="300"
+      height={mobile ? 330 : 235}
+      fontFamily="Consolas"
+      font-size="10pt">
+      <Window.Content scrollable>
+        <Section>
+          {disposalpipes.map(disposalpipe => {
+            return (
+              <Flex key={disposalpipe.disposaltype} justify="space-between" align="stretch" style={{ "border-bottom": "1px #555 solid" }}>
+                <Flex.Item direction="row">
```

Comment:
I'm sure this does *something*, but I would imagine `direction` should be on the `Flex` rather than the `Flex.Item`s?

Reply:
Removed, remnant from vending tgui. Direction row is default for flex